ffmpeg-plugin/configure-ffmpeg.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2024 Intel Corporation
+
+set -e
+
+cd FFmpeg
+export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
+pkg-config --exists --print-errors libmcm_dp
+
+# copy source files to allow the configure tool to find them
+#cp -f ../mcm_* ./libavdevice/
+
+./configure --enable-shared --enable-mcm
+cd ..
```

Comment:
I would advice not to use constructs like `export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig`, this will overwrite the variable from user bash session and can break his future build efforts also can have different location then the one hardcoded here.

```suggestion
cd "${SCRIPT_DIR}/FFmpeg"
export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
pkg-config --exists --print-errors libmcm_dp

# copy source files to allow the configure tool to find them
# cp -f ../mcm_* ./libavdevice/

./configure --enable-shared --enable-mcm
cd "${OLDPWD}"
```

Reply:
For any bash script that is used in process - it is not advised to use relative paths as you can not be sure from where the script is being lunched. This can lead to undefined behavior with removing user files as the worst case scenario.

Bash defines global variables that can be used when needed. So here for example I encourage you to use `"${OLDPWD}"` after doing `cd "${SCRIPT_DIR}/FFmpeg"` earlier. The best approach would be not to use `cd` in scripts at all, but instead lunch commands with directory name, for example the `git` allows usage of `git -C <dir>`

Reply:
I like your ideas. Let's improve the script in a separate PR. I suggest not to amend this one.